### DISCUSSION TITLE
Implement Markdown Alerts

### DISCUSSION THREAD

**Discussion by gianfa** [quarto-dev/quarto-cli#9573] [Q&A] [answered]
### Description

The relatively new [Markdown Alerts](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts) feature would bring more graphically pleasing and more functional handling to the highlighted text

**Comment by mcanouil** [comment, marked as answer]
It's callouts blocks, see <https://quarto.org/docs/authoring/callouts.html>.

> more graphically pleasing and more functional handling to the highlighted text

Compared to what and for which format?
<img width="1624" alt="image" src="https://github.com/quarto-dev/quarto-cli/assets/8896044/89c8703b-2e66-45f9-a6ee-528143156959">

**Reply by gianfa** [reply]
I meant comparing to simple quotes style.
I wasn't able to find it, thank you!

So this feature request is already solved :)

**Comment by plbarrio** [comment]
If you still need to use GitHub alert syntax (> [!NOTE]) with Quarto, I built a small Lua filter that bridges the conversion automatically to native Quarto callouts:

https://github.com/plbarrio/alerts-normalize
